apps/web/src/state/info/queries/tokens/priceData.ts

```diff
@@ -41,12 +41,17 @@ const fetchTokenPriceData = async (
 }> => {
   // Construct timestamps to query against
   const endTimestamp = getUnixTime(new Date())
-  const timestamps = []
+  let timestamps = []
   let time = startTimestamp
   while (time <= endTimestamp) {
     timestamps.push(time)
     time += interval
   }
+  if (chainName === 'BSC' && !checkIsStableSwap()) {
+    const latestBlock = await getBlocksFromTimestamps([endTimestamp], 'asc', 500, chainName)[0]
+    timestamps = timestamps.filter((t) => t < latestBlock - 32) // nodeReal will sync the the 32 block before latest
```

Comment:
Isn't `latestBlock` type of `Block`?

Reply:
I had update the code, this version has some issue